docs/Developer Reference/getting-help.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: nodes.liquid
+section: smartContract
+date: Last Modified
+title: "Getting Support"
+permalink: "docs/getting-help/"
+---
+## Where do I go to get help and support? 
+
+If you run into issues and the available documentation, videos, and code repositories are not able to assist you, the best way to get help is to follow the support escalation process in this document. Sometimes you might have a question that is too theoretical or hasn't been solved, so you might not always get what you're looking for!
+
+## 1. Double check the documentation
+
+Check to see if you missed any code, documentation, blog, or video on the topic or issue you're looking for. There are typically a few different resources on a topic if one doesn't answer exactly what you're looking for. You can also use the documentation search bar to look up things as well. 
+
+## 2. Do a web search for the specific error or situation you're in
+
+Often someone else has asked the same question that you're asking. If you copy and paste the error into the Google or web search bar, there is a good chance that you will find some helpful material from someone else who has already found the solution to your question.
+
+## 3. Open an issue on the github or code repository
+
+This is only applicable if you're working with a certain set of code. For example, if you're having an issue working with the [Chainlink Hardhat Starter kit](https://github.com/smartcontractkit/hardhat-starter-kit), open an issue on the repo explaining exactly what's going on and someone might have the answer that you need.
+
+When writing issues, remember to:
+- Keep titles short
+- Be clear and concise about the issue that you are encountering
+- Format your issue description. Use [three backticks (```)](https://discuss.codecademy.com/t/using-backticks-to-format-your-code/3697) to format format your code or log output.
```

Comment:
```suggestion
- Format your issue description. Use [three backticks (```)](https://www.freecodecamp.org/news/how-to-format-code-in-markdown/#code-blocks) to format format your code or log output.
```